playground/src/components/channels-list/channels-list.js

```diff
@@ -54,8 +52,7 @@ export class ChannelsList extends React.Component {
         return value ? (
           <Constraints.Horizontal constraint="m">
             <Text.Wrap>
-              {value[this.props.applicationState.project.dataLocale] ||
-                NO_VALUE_FALLBACK}
+              {value[this.props.dataLocale] || NO_VALUE_FALLBACK}
```

Comment:
I think this should be  `this.props. applicationContext.dataLocale`. Or we could add a mapping function to `withApplicationContext` which might be even nicer!

Reply:
Ups, thanks!